### DISCUSSION TITLE
Warning with EXPERIMENTAL_API cannot be disable

### DISCUSSION THREAD

**Discussion by Hardel-DW** [nextauthjs/next-auth#5833] [Ideas]
### Description 📓

I'm using Next Auth with Next JS 13 to try out the new features. and so I am using the "unstable_getServerSession" function from NextAuth.

Currently, on every hot reload of NextJS, the warning message is sent every time.  
It's cumbersome and impractical.

My suggestion is simple here are several proposals:
- To be able to deactivate these warnings.
- This warning only occurs at startup.

![image](https://user-images.githubusercontent.com/42413417/201810308-a009ed7c-1601-45ef-8a8d-6add37010bc9.png)

### How to reproduce ☕️

Use "unstable_getServerSession" with Server Components.

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by nachthammer** [comment]
Can we please have a look at this issue. It should be almost no work to do this (if you give me a little guidance through the project i can do it myself) and it would enhance the logging of big projects a lot. Maybe in general just display this message on start up like it is done with the experimental feature message of NextJS when using server actions.
